packages/itinerary-body/src/styled.tsx

```diff
@@ -594,6 +594,12 @@ export const StepStreetName = styled.span`
   font-weight: 500;
 `;
 
+export const StepLength = styled.span`
+  font-weight: 200;
+  opacity: 0.8;
```

Comment:
Color contrast is getting flagged here.  Maybe keep the opacity at 100%, and instead use `|` to provide some separation in the line? So it would read:
Left on Pioneer Sq N (path) | 22 feet | (Mapillary Btn)

Reply:
Ugh that's really annoying... I initially had the `|` but it looks really really messy. Can we get away with 0.9?

Reply:
.9 does pass! Visually I'm not sure there's clear distinction between the opacities though. Maybe an `-` instead of `|`? or would that have the same issue?

Reply:
I think that's even more confusing... Let me find the lowest opacity that still passes AA. Or are we targeting AAA?

Reply:
Nope AA is fine! I think it'd be somewhere between `.8` and `.9`

Reply:
`0.8975` 😵‍💫 (fbb2294ee85649a3b1d9ee3fec22da917411e9bb)